src/Handler/CommandNameExtractor/IdentifierMapExtractor.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace League\Tactician\Handler\CommandNameExtractor;
+
+/**
+ * Extract the an identifier based on the given map where the key of the map is the FQCN for the Command.
+ */
+class IdentifierMapExtractor implements CommandHandlerNameExtractor
+{
+    /**
+     * An identity map that maps the FQCN of a Command (as key) to an identifier (as value).
+     *
+     * @var string[]
+     */
+    private $identityMap = [];
+
+    /**
+     * Registers the identity map with this Extractor.
+     *
+     * @param string[] $identityMap
+     */
+    public function __construct(array $identityMap)
+    {
+        $this->identityMap = $identityMap;
```

Comment:
@mvriel 

Should we assert here that the keys are

* `string`s
* map to names of existing classes

?